src/components/TempTodo.tsx

```diff
@@ -0,0 +1,42 @@
+import { Todo } from '../types';
+
+type Props = {
+  tempTodo: Todo,
+  handleDeleteTodo: (value: number) => void,
+};
+
+export const TempTodo: React.FC<Props> = ({ tempTodo, handleDeleteTodo }) => {
+  return (
+    <li
+      className="todo"
+      key={tempTodo.id}
+    >
+      <label className="todo__status-label">
+        <input
+          type="checkbox"
+          className="todo__status"
+          checked={false}
```

Comment:
The checkbox should reflect the current status of the todo item. Use tempTodo.completed for checked attribute instead of hardcoded false.